label_sync/Makefile

```diff
@@ -12,48 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: build test
+all: bazel
 
 
-LABEL_SYNC_VERSION       ?= 0.1
-
+# TODO(fejta): deploy via bazel
 # These are the usual GKE variables.
 PROJECT       ?= k8s-prow
-BUILD_PROJECT ?= k8s-prow-builds
 ZONE          ?= us-central1-f
 CLUSTER       ?= prow
 
-# Build and push specific variables.
-REGISTRY ?= gcr.io
-PUSH     ?= gcloud docker -- push
-
-DOCKER_LABELS=--label io.k8s.prow.git-describe="$(shell git describe --tags --always --dirty)"
+bazel:
+	@echo Use bazel foo or ./planter/planter.sh bazel foo
+	@echo "  bazel build //label_sync/...  # build"
+	@echo "  bazel test //label_sync/...  # test"
+	@echo "  bazel run //label_sync:push  # push image"
+	@echo "  bazel run //label_sync:deploy # deploy"
+	@exit 1
 
 get-cluster-credentials:
 	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
 
-get-build-cluster-credentials:
-	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(BUILD_PROJECT)" --zone="$(ZONE)"
-
-build:
-	go install ./...
-
-test:
-	go test -race -cover $$(go list ./... | grep -v "\/vendor\/")
-
 .PHONY: build test get-cluster-credentials
```

Comment:
Remove `build` and `test` targets from `.PHONY` target.

Reply:
Done